ResearchKitTests/ORKTextChoiceCellGroupTests.m

```diff
@@ -55,6 +55,19 @@ - (NSArray *)textChoices {
     return choices;
 }
 
+- (NSArray *)textChoicesWithExclusive {
+    static NSArray *choicesWithExclusive = nil;
+    
+    if (choicesWithExclusive == nil) {
+        choicesWithExclusive = @[[ORKTextChoice choiceWithText:@"choice 01" value:@"c1"],
+                                [ORKTextChoice choiceWithText:@"choice 02" value:@"c2" exclusive:YES],
+                                [ORKTextChoice choiceWithText:@"choice 03" value:@"c3"],
+                                [ORKTextChoice choiceWithText:@"choice 04" value:@"c4"]];
```

Comment:
Given the way you've done the API, you should also test the case wehre there are *two* exclusive choices. Or even *all* exclusive choices.